src/Main.hs

```diff
@@ -85,15 +84,15 @@ generateRay :: Camera -> Int -> Int -> Int -> Int -> Ray
 generateRay camera width height x y = 
     let 
          -- Vector from camera to lookAt point
-        eyeVector = buildEyeVector camera 
+        centerVector = eyeVector camera
         -- Vector in local right direction
-        rightVector = normalize (eyeVector `cross` Vector 0 1 0)
-        upVector = normalize (rightVector `cross` eyeVector)
+        rightVector = normalize (centerVector `cross` Vector 0 1 0)
+        upVector = normalize (rightVector `cross` centerVector)
         -- Halves are taken to make right angles
-        halfFov = fovInRadians camera
+        halfFov = fov camera / 2
```

Comment:
I know operator precedence takes care of this for us, but would parens be more clear? What's the convention?